pymongo/collection.py

```diff
@@ -2435,16 +2435,10 @@ def watch(self, pipeline=None, full_document='default', resume_after=None,
         .. _change streams specification:
```

Comment:
For the `:Parameters:` docs above add:
```
          - `start_at_operation_time` (optional): If provided, the resulting
            change stream will only return changes that occurred at or after
            the specified :class:`~bson.timestamp.Timestamp`. Requires
            MongoDB >= 4.0.

#etc...

        .. versionchanged:: 3.7
           Added the ``start_at_operation_time`` parameter.
        .. versionadded:: 3.6
```

Reply:
I added a different description for this option, but I think yours is slightly clearer to understand, albeit a bit verbose. Going with yours!